frontend-web/webclient/app/UCloud/FilesApi.tsx

```diff
@@ -636,10 +655,15 @@ class FilesApi extends ResourceApi<UFile, ProductStorage, UFileSpecification,
                         cancelButtonColor: "primaryMain",
                         confirmButtonColor: "errorMain",
                         onConfirm: async () => {
-                            await cb.invokeCommand(
-                                this.emptyTrash(bulkRequestOf({id: cb.directory?.id ?? ""}))
-                            );
-                            cb.navigate("/drives");
+                            try {
+                                await cb.invokeCommand(
+                                    this.emptyTrash(bulkRequestOf({id: cb.directory?.id ?? ""}))
+                                );
+                                cb.navigate("/drives");
```

Comment:
Would it be possible to change this such that it simply sends you back to the root of the drive instead of the drive view? I always found it a bit weird that we go all the way back to the drive listing.

Reply:
I was under the impression that we had discussed this a while ago and decided not to do it.

Reply:
In that case we might have had a reason. Do you remember why we decided not to?

Reply:
Jonas stated yesterday that it might be due to us not knowing how or where a provider does trash so by going to drives it was a safe bet.

Reply:
In that case, we can definitely change to go to the root of the drive.

Reply:
Something like this should work, I think. But maybe done prettier.
`cb.navigate(AppRoutes.files.path(pathComponents(cb?.directory)[0]));`